examples/example_hmc_vault.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# HMC vault file for zhmcclient end2end tests
+#
+# This file defines the credentials for real HMCs for the purpose of end2end
+# tests against these HMCs.
+# The file must have the format defined in
+# zhmcclient/testutils/hmc_vault_file.py of the python-zhmcclient project.
+#
+# HMC vault files conform to the format of Ansible vault files in YAML
+# format and define specific variables for HMC authentication.
+#
+# Brief description of the file format:
+#
+#   hmc_auth:
+#     <hmc_name>:  # DNS hostname, IP address, or nickname of HMC
+#       hmc_userid: <hmc_userid>
+#       hmc_password: <hmc_password>
+#       hmc_verify: <hmc_verify>
+#       hmc_ca_certs: <hmc_ca_certs>
```

Comment:
The auth parameters need to be named so they are compatible with the hmc_auth parameter of the zhmc Ansible modules, i.e. `userid`, `password`, `verify`, `ca_certs`.

Reply:
done